CHANGES

```diff
@@ -3,13 +3,18 @@ big Changelog
 
 Here you can see the full list of changes between each big release.
 
-Version 0.8.5
+Version 0.9.0
```

Comment:
Don't forget to update version in `gradle.properties`

Reply:
Done.